vsphere/virtual_machine_config_structure.go

```diff
@@ -262,6 +262,12 @@ func schemaVirtualMachineConfigSpec() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "The UUID of the virtual machine. Also exposed as the ID of the resource.",
 		},
+		"latency_sensitivity": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "normal",
```

Comment:
Could use the govmomi generated const here: `types.LatencySensitivitySensitivityLevelNormal`